contributing_to_docs/doc_guidelines.adoc

```diff
@@ -257,6 +257,31 @@ link:https://redhat-documentation.github.io/modular-docs/#creating-procedure-mod
 When needed, use `.Prerequisites`, `.Next steps`, or `.Additional resources` syntax to suppress TOC formatting within a module. Do not use `==` headings or xrefs in modules.
 ====
 
+[id="using-conscious-language"]
+== Using conscious language
+
+To assist with the removal of the problematic word "master" from the documentation, use the following terminology when referring to OpenShift control plane nodes:
+
+[options="header"]
+|===
+|Branch |Control plane node reference
+
+|`main` and `enterprise-4.9`
+|Control plane node
+
+|`enterprise-4.8` and earlier
+|Control plane (also known as master) node
+
+|`enterprise-3.11`
+|Master node
+
+|===
+
+[NOTE]
+====
+You can replace "node" in the preceding examples with "machine", "host", or another suitable description.
+====
+
```

Comment:
After speaking with @vikram-redhat can we add something like the following note? This is to clarify that if this is the first time you're introducing "control plane" into content, then that's when you need to do the manual cherry picks/PRs for older versions.

```
If you’re cherry picking a PR from `main` to an older version (4.8 and earlier), and this is the first time that the term “control plane” is being introduced in the module/assembly, then do manual cherry picks back to the older versions to add in the “(also known as master)” phrasing.
```

@vikram-redhat Can you please review to make sure that this sounds right from what we discussed the other day?

Reply:
@bergerhoffer thank you for your further review. I have now added the note to the PR.

Reply:
Thanks, LGTM!

Reply:
LGTM

Reply:
Great, thank you!

Reply:
I am going to create a script to update "control plane node (also known as the master node)" in the docs to "Control plane (also known as master) node", so that it is in-line with these guidelines. I will create a separate series of PRs for `enterprise-4.8` and earlier for that.